src/generators/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './AbstractRenderer';
 export * from './java';
 export * from './javascript';
 export * from './typescript';
+export * from './go';
```

Comment:
As in above comments.

Reply:
If I do this, the changes I did here are not needed, right? https://github.com/asyncapi/modelina/pull/267#discussion_r659729267

Reply:
Yes :)

Reply:
Oki doki! fixed then :)